spec/stepmod/converters/eqn_spec.rb

```diff
@@ -80,4 +80,19 @@
       expect(converter.convert(input)).to eq(output)
     end
   end
+
+  context 'when sup/sub tags used' do
+    let(:xml_input) do
+      <<~XML
```

Comment:
Can we use heredoc/multi-line strings for comparison?

Reply:
Done